mypy/errors.py

```diff
@@ -720,6 +722,8 @@ def generate_ignore_without_code_errors(
                 False,
                 False,
                 False,
+                origin=(self.file, None),
+                target=self.target_module,
```

Comment:
**Question 1:** I'm not sure if `origin` is required here. I noticed that the other places that passed in `target=` passed this too, so I assumed it couldn't hurt. Should I remove it?

**Question 2:** I would have liked to use `target=self.current_target()` here, similar to the other uses in the file, but as I detailed in another comment that causes an error. Is this good enough? Are we losing important context by taking this approach?